packages/core/scss/global/elements/_elements.lists.scss

```diff
@@ -62,3 +62,18 @@ ol > li {
     font-size: 16px;
   }
 }
+blockquote > ol > li {
+  &:before {
+    position: relative;
+    left: -5px;
+    margin-right: 5px;
+  }
+}
+@media only screen and (max-width: 600px) {
+  blockquote > ol > li {
+    &:before {
+      left: -55px;
+      position: absolute;
+    }
+  }
+}
```

Comment:
For å bevare avstanden mellom listepunkt og tekstinnhold foreslår jeg å heller justere margin/padding på `ol` og `ul`-elementer.

Reply:
Det var et veldig godt råd, takk!